checker/src/main/java/org/checkerframework/checker/immutability/ExtendedViewpointAdapter.java

```diff
@@ -0,0 +1,14 @@
+package org.checkerframework.checker.immutability;
+
+import org.checkerframework.framework.type.AnnotatedTypeMirror;
+import org.checkerframework.framework.type.ViewpointAdapter;
+
+import javax.lang.model.element.AnnotationMirror;
+
+public interface ExtendedViewpointAdapter extends ViewpointAdapter {
```

Comment:
We really need some documentation for what this interface is for. Is it PICO specific? Should it just be in the main interface?

Reply:
@wmdietl I removed those dummy interfaces in [2789ede](https://github.com/opprop/immutability/pull/43/commits/2789edea849c3897ff0ec2c0524b8127fa3da20a)